web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -247,19 +278,36 @@
                   ? `${AppRoute.ALBUMS}/${albumId}`
                   : AppRoute.PHOTOS}"
                 on:click={() => dispatch('closeViewer')}
+                on:contextmenu|preventDefault={() => handleRightClickPeople(index)}
               >
                 <div class="relative">
-                  <ImageThumbnail
-                    curve
-                    shadow
-                    url={api.getPeopleThumbnailUrl(person.id)}
-                    altText={person.name}
-                    title={person.name}
-                    widthStyle="90px"
-                    heightStyle="90px"
-                    thumbhash={null}
-                    hidden={person.isHidden}
-                  />
+                  <div class="absolute">
+                    <ImageThumbnail
+                      curve
+                      shadow
+                      url={api.getPeopleThumbnailUrl(person.id)}
+                      altText={person.name}
+                      title={person.name}
+                      widthStyle="90px"
+                      heightStyle="90px"
+                      thumbhash={null}
+                      hidden={person.isHidden}
+                    />
+                  </div>
+                  {#if showActionPerson[index]}
+                    <button
+                      class="absolute rounded-xl items-center bg-gray-300 dark:bg-slate-100 py-3 px-6 text-left text-sm font-medium text-immich-fg hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-inset dark:text-immich-dark-bg dark:hover :bg-gray-200 transition-colors"
+                      use:clickOutside
+                      on:outclick={() => (showActionPerson[index] = false)}
+                      on:click|preventDefault|stopPropagation={() => handleHidePerson(index)}
+                    >
+                      {#if person.isHidden}
+                        Unhide
+                      {:else}
+                        Hide
+                      {/if}
```

Comment:
Maybe an eye icon or something for the "emojification-factor"? ;)

Reply:
I thought about that, but I don't know what look the best between having a text or an icon for that one :


| | Hide | Unhide |
|  :---: | :---: | :---: |
| Icons | ![Screenshot from 2023-12-29 12-42-15](https://github.com/immich-app/immich/assets/74269598/43905451-735c-46d2-abb1-070aa137864e) | ![Screenshot from 2023-12-29 12-42-08](https://github.com/immich-app/immich/assets/74269598/7c2838a1-af3b-46c7-b119-71a6e2136229) |
| Text | ![Screenshot from 2023-12-29 12-42-38](https://github.com/immich-app/immich/assets/74269598/d16f5cbc-38e9-4462-858c-7b0766679235) | ![Screenshot from 2023-12-29 12-42-29](https://github.com/immich-app/immich/assets/74269598/7399b1ba-5637-4869-99db-276e3a707bb4) |

Reply:
Hm yeah you're right... In my imagination it looked better lol